docs/app/Components/ComponentDoc/ComponentProps/ComponentProps.js

```diff
@@ -50,7 +50,7 @@ export default class ComponentProps extends Component {
         />
 
         {activeName && (
-          <div>
+          <div style={{ overflowX: 'auto' }}>
```

Comment:
Please extract this style to variable like [there](https://github.com/redbmk/Semantic-UI-React/blob/ea316aee9dc35c6a1086459323555ace9ad76701/docs/app/Components/ComponentDoc/ComponentProps/ComponentPropsHeader.js#L8). Double curly braces are performance killer as they always return the new object on each render

Reply:
No problem. I went ahead and moved that into a `const`